package.json

```diff
@@ -27,81 +27,81 @@
     "@tableflip/react-dropdown": "^1.1.1",
     "@tableflip/react-inspector": "^2.3.0",
     "brace": "^0.11.1",
-    "chart.js": "^2.7.2",
+    "chart.js": "^2.7.3",
     "d3": "^5.7.0",
     "details-polyfill": "^1.1.0",
     "file-extension": "^4.0.5",
     "filesize": "^3.6.1",
-    "i18next": "^11.6.0",
-    "i18next-browser-languagedetector": "^2.2.3",
+    "i18next": "^11.10.2",
+    "i18next-browser-languagedetector": "^2.2.4",
     "i18next-icu": "^0.6.0",
     "i18next-xhr-backend": "^1.5.1",
-    "internal-nav-helper": "^1.0.2",
+    "internal-nav-helper": "^1.1.0",
     "ipfs-css": "^0.10.0",
     "ipfs-geoip": "^2.3.0",
-    "ipfs-redux-bundle": "^4.1.1",
-    "ipfs-unixfs": "^0.1.15",
-    "ipld": "^0.17.3",
+    "ipfs-redux-bundle": "^4.1.2",
+    "ipfs-unixfs": "^0.1.16",
+    "ipld": "^0.20.1",
     "ipld-explorer-components": "^1.0.0",
     "is-binary": "^0.1.0",
-    "is-ipfs": "^0.4.2",
+    "is-ipfs": "^0.4.8",
     "milliseconds": "^1.0.3",
-    "money-clip": "^2.1.0",
-    "multiaddr": "^5.0.2",
+    "money-clip": "^3.0.0",
```

Comment:
Note to self: This is a major, but from the changelog it looks like it was a packaging issue https://github.com/HenrikJoreteg/money-clip#change-log ... is fine.

Reply:
Yeah, it only has [minimal changes](https://github.com/HenrikJoreteg/money-clip/commit/0118bbabcd6f4108ab279cb18fb45b35d701b3c2) that don't affect our use (`import { getConfiguredCache } from 'money-clip'` in `bundle-cache.js`)